.github/workflows/docs_deployment.yml

```diff
@@ -0,0 +1,31 @@
+name: Deploy docs website
+
+on:
+  push:
+    branches: [main, release]
```

Comment:
Interesting, maybe should be only release? As it Is the released code?

Reply:
We're doing two different deployments according to the branch that's being pushed to. I think it's good to test doc deployment with the main branch before we do prod deployment.

Reply:
You are right! Great!